Content/ContentTypeResolver/SingleSnippetSelectionResolver.php

```diff
@@ -81,18 +72,13 @@ public function resolve($data, PropertyInterface $property, string $locale, arra
             return new ContentView(null, ['id' => null]);
```

Comment:
would be nice to use the `DefaultSnippetManager::loadIdentifier()` method in this class too